lib/rtcpeerconnection/chrome.js

```diff
@@ -38,6 +38,7 @@ function ChromeRTCPeerConnection(configuration, constraints) {
   util.interceptEvent(this, 'datachannel');
   util.interceptEvent(this, 'signalingstatechange');
 
+  sdpFormat = sdpFormat || sdpUtils.getSdpFormat(newConfiguration.sdpSemantics);
```

Comment:
if you are planning to add an option to let users choose sdpFormat by specifying in connectOptions as mentioned in the description then I am not sure how caching would work. What if they want to create two rooms using different sdpFormat from single tab?

Reply:
@makarandp0 This flag is internal only and not part of the public API. So we will always default to plan-b until we decide to support unified plan.

Reply:
@manjeshbhargav, Looking again maybe it's not worth caching sdpFormat here,  since its cached at sdpUtils.getSdpFormat anyways. 

My concern is we may endup reusing Twilio-Video for multiple sessions (like in unit test or inside our app) and may want to change the `sdpSemantics ` value across the sessions.